modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml

```diff
@@ -170,11 +188,21 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ExceptionBody'    
+                $ref: '#/components/schemas/ExceptionBody'
+        '409':
+          description: |-
+            Possible issues: 
+            - Multiple objects are created/updated at the same time.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionBody'
         '500':
           description: |-
             Possible issues:
             - The provided TED API key cannot be currently validated.
+            - The CVS service is not available to validate the notice.
+            - CVS service failed to validate the notice.
```

Comment:
This might be understood as "The CVS service reported errors in the validation report", or as "The CVS service was not able to execute the validation" because of some problem.
So it should be reformulated to make it clearer which it is.

Reply:
OK